specification/hugr.md

```diff
@@ -1212,7 +1206,9 @@ descended from nodes in $S$ (i.e. reachable from $S$ by following hierarchy edge
 including $S$ itself.
 
 Call two nodes $a, b \in \Gamma$ _separated_ if $a \notin \\{b\\}^\*$ and
-$b \notin \\{a\\}^\*$ (i.e. there is no hierarchy relation between them).
+$b \notin \\{a\\}^\*$ (i.e. there is no hierarchy relation between them). Note
+that this is much the same requirement as convexity, but following the hierarchy
```

Comment:
I don't really see why separatedness is the same (or similar) to convexity. Certainly convex subgraphs of the hierarchy tree need not be separated.

Reply:
(Separatedness now removed/inlined)